common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -1204,3 +1205,114 @@ def _cross_domain_post(self, csrf_cookie):
             HTTP_REFERER=self.REFERER,
             HTTP_X_CSRFTOKEN=csrf_cookie
         )
+
+
+@ddt.ddt
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class UnenrollmentTest(EnrollmentTestMixin, ModuleStoreTestCase):
+    """
+    Tests unenrollment functionality. The API being tested is intended to
+    unenroll a learner from all of their courses.g
```

Comment:
Nit: typo.

Reply:
This is my signature typo. A trailing g. At least this time I didn't add it to actual code... ;)